src/redux/slices/books/bookSlice.js

```diff
@@ -0,0 +1,54 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { createSlice } from '@reduxjs/toolkit';
+
+// const bookSlice = createSlice({
+//   name: 'book',
+//   initialState: [],
+//   reducers: {
+//     deleteBook: (state, action) => {
+//       const bookTitle = action.payload;
+//       state.myBooks = state.myBooks.filter((book) => book.title !== bookTitle);
+//     },
+//     addBook: (state, action) => {
+//       const book = action.payload;
+//       state.myBooks.push(book);
+//     },
+//   },
+// });
+
```

Comment:
- [ ] Kindly remove the commented piece of code to make your project follow the JavaScript best practices 👍